src/components/Quiz.js

```diff
@@ -43,7 +50,7 @@ const Quiz = ({ questions, weights, updateWeights, setIsQuizComplete }) => {
     if (areAllQuestionsAnswered) {
```

Comment:
I didn't see this some where and this is completely a coding preference but areAllQuestionsAnswered might be better named something like validateQuestionsAnswered or something similar. No need to change it right now.